draft-ietf-taps-interface.md

```diff
@@ -887,12 +887,27 @@ over these options.
 Name:
 : multipath
 
-This property specifies whether an application considers it useful to
-transfer data across multiple paths between the same end hosts. Generally,
-in most cases, this will improve performance (e.g., achieve greater throughput).
-One possible side-effect is increased jitter, which may be problematic for
-delay-sensitive applications. The recommended default is to Ignore this option.
+Type:
+: Enumeration
```

Comment:
This removes the possibility to prefer vs require multipath functionality.
I would rather make it a set of tuples like the Interface type:
```suggestion
:  Set (Preference, Enumeration)
```

Reply:
Yes, I think this will also be more consistent.

Reply:
That is a design choice we can make, but then we should be explicit abut it. Maybe "If it is set to something except disable, a taps system should prefer protocols that provide the chosen mode of operation.".